source/_drafts/Video-js-5-s-fluid-mode-and-playlist-picker.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Video.js 5's fluid mode and playlist picker
+tags:
+  - videojs 5
+  - fluid
+---
+
+## How it works
+In videojs 5.0, we added support for truely fluid layouts with video.js.
+You can see an example of it on the [videojs website][website].
+
+It is done by using [intrinsic ratios][ratios]. In short, you make the video element take up 100% `width` and make the `height` be `0`.  We can set the player `width` to a particular size, say 50%; this way, as we resize the the browser window, the player should stay 50% the same of the window but keep it's aspect ratio for the video. The main trick is that in addition to setting the `width` on the player, we also apply a padding. Specifically, we want to set the following:
```

Comment:
I'm not sure the sentence explaining how intrinsic ratios in CSS works is clarifying the issue. I might suggest the link for understanding how they work then jump right to upshot.